gmpydl/Dockerfile

```
@@ -1,20 +1,32 @@
-FROM python:2-slim
-
-RUN set -eux; \
-	apt-get update; \
-	apt-get install -y --no-install-recommends \
-		wget \
-	; \
-	rm -rf /var/lib/apt/lists/*
+FROM python:2-alpine
```

Generally this looks good, but I think we should probably pin this more explicitly to avoid surprises:

```suggestion
FROM python:2-alpine3.10
```